tests/integration/issuing.test.js

```diff
@@ -122,13 +139,20 @@ describe('Verifiable Credential issuance where issuer has a Dock DID', () => {
     );
   }, 40000);
 
-  test('(JWT) Issue a verifiable credential with ed25519 key and verify it', async () => {
+  test('(JWT) Issue a verifiable credential with ed25519 key and verify it (2018)', async () => {
     const issuerKey = getKeyDoc(issuer1DID, dock.keyring.addFromUri(issuer1KeySeed, null, 'ed25519'), 'Ed25519VerificationKey2018');
     const credential = await issueCredential(issuerKey, unsignedCred, true, null, null, null, null, false, 'jwt');
     const result = await verifyCredential(credential, { resolver });
     expect(result.verified).toBeTruthy();
   }, 40000);
 
+  test('(JWT) Issue a verifiable credential with ed25519 key and verify it (2020)', async () => {
+    const issuerKey = getKeyDoc(issuer1DID, dock.keyring.addFromUri(issuer1KeySeed, null, 'ed25519'), 'Ed25519VerificationKey2020');
+    const credential = await issueCredential(issuerKey, unsignedCred, true, null, null, null, null, false, 'jwt');
+    const result = await verifyCredential(credential, { resolver });
+    expect(result.verified).toBeTruthy();
+  }, 40000);
+
```

Comment:
Can you add a test for presentation as well with this new scheme?

Reply:
done